content/repositories/releasing-projects-on-github/managing-releases-in-a-repository.md

```diff
@@ -79,7 +79,13 @@ If you @mention any {% data variables.product.product_name %} users in the notes
 
 {% cli %}
 
-Releases cannot currently be edited with {% data variables.product.prodname_cli %}.
+1. To edit a release, use the `gh release edit` subcommand. Replace `tag` with the tag representing the release you wish to edit. For example, to edit a given release's title:
```

Comment:
```suggestion
1. To edit a release, use the `gh release edit` subcommand. Replace `TAG` with the tag representing the release you wish to edit. For example, to edit a given release's title, use the following code:
```

Just a couple small tweaks for ease of localization, and to adhere to our style guide.